### DISCUSSION TITLE
Opening a specific page via cli

### DISCUSSION THREAD

**Discussion by zevisvei** [kiwix/.github#9] [Q&A]
How can I open a certain page in Kiwix for a Windows desktop?
I saw [here](https://wiki.kiwix.org/wiki/FAQ) that this is the command:
`kiwix.exe -articleByUrl "zim://A/foo.html" bar.zim`
But somehow it doesn't work

**Comment by kelson42** [comment]
This has not been implemented in version 2 of Kiwix Desktop. Maybe this should be considered as a bug. Do you have a use case needing this feature?

**Reply by zevisvei** [reply]
Yes, there are many cases where I want to save a shortcut to immediately open a certain page.

**Reply by kelson42** [reply]
I have created issue at https://github.com/kiwix/kiwix-desktop/issues/1161. It will be implemented at somepoint but I can not say when. I will close this discussion.
